index.js

```diff
@@ -44,10 +44,13 @@ function getPackageJson(packageName) {
   try {
     return require(`${packageName}/package.json`);
   } catch (requireError) {
-    if (requireError.code !== "ERR_PACKAGE_PATH_NOT_EXPORTED")
+    if (requireError.code === "MODULE_NOT_FOUND") {
+      throw requireError;
+    } else if (requireError.code !== "ERR_PACKAGE_PATH_NOT_EXPORTED") {
```

Comment:
Can you remove the `else`? It's useless after the `throw`. Otherwise, LGTM :-)

Reply:
Sure, you're absolutely right about else :)

Reply:
Done!